regimes/mx/fuel_complement.go

```diff
@@ -0,0 +1,60 @@
+package mx
+
+import (
+	"github.com/invopop/gobl/cal"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/num"
+	"github.com/invopop/gobl/org"
+	"github.com/invopop/gobl/tax"
+)
+
+// FuelComplement carries the data to produce a CFDI's "Complemento de Estado de
+// Cuenta de Combustibles para Monederos Electrónicos" (version 1.2 revision B)
+// providing detailed information about fuel purchases made with electronic
+// wallets. In Mexico, e-wallet suppliers are required to report this
+// complementary information in the invoices they issue to their customers.
+type FuelComplement struct {
```

Comment:
So after reading the official name, had you considered any alternative names? Perhaps `EWalletFuelComplement`? I'm worried about potential colisions, and the fact that "FuelComplement" doesn't cover the wallet part...

Reply:
Considering this name will also be part of the schema name, I tried to make it as short as possible. I suppose future name collisions are indeed a possibility (although probably small). In the XML, they call the root tag `EstadoCuentaCombustibles`, so they give more priority to the "estado de cuenta" part of the name than to the "e-wallet" aspect. So here's a list of options with their trade-offs:

Name | Merits | Drawbacks
--- | --- | ---
`FuelComplement` | short and to the point | not fully meaningful, collision-able |
`EWalletFuelComplement` | meaningful, no collisions | long schema name |
`FuelAccountStatement` | direct translation of `EstadoCuentaCombustibles` | no mention of e-wallet, no mention of being a complement

I don't love any of them, but I think all of them would work. Let me know if you have any preference (or any other suggestion) and we'll go with that.

Reply:
I'd be inclined to go with the direct translation: `FuelAccountStatement` or `FuelAccountBalance` which is the translation I got from ChatGPT.

Reply:
hehe, I got "Account Statement" from Deepl, but I like "Account Balance" better! (AIs FTW!)